src/classes/CW_MarcomBriefViewController.cls

```diff
@@ -0,0 +1,313 @@
+/**
+* File Name   :    CW_MarcomBriefViewController.cls
+* Description :    B-109868: Add Related objects to the MarCom Brief View 
+*
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 04/06/2014 Cristina P.	Intial
+* 0.2 30/06/2014 Cristina P.	B-120546: Change Account Partner Privileges
+*
+*/
+public without sharing class CW_MarcomBriefViewController 
```

Comment:
* This class seems to have grown very large and is becoming difficult to maintain. I suggest we consider refactoring to reduce the number of methods.
* Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.